src/gluonts/ev/usage_example.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from toolz import take
+
+from gluonts.dataset.split import TestTemplate, OffsetSplitter
+from gluonts.dataset.repository.datasets import get_dataset
+from gluonts.model.npts import NPTSPredictor
+from gluonts.ev.api import MultiMetricEvaluator, get_evaluation_data_batches
+from gluonts.ev.metrics import Coverage, MeanSquaredError, QuantileLoss
+
+dataset = get_dataset("electricity")
+
+prediction_length = dataset.metadata.prediction_length
+freq = dataset.metadata.freq
+
+dataset_test = list(take(10, dataset.test))
+
+test_template = TestTemplate(
+    dataset=dataset_test, splitter=OffsetSplitter(offset=-prediction_length)
+)
+
+test_data = test_template.generate_instances(
+    prediction_length=prediction_length
+)
+
+predictor = NPTSPredictor(prediction_length=prediction_length, freq=freq)
+forecast_it = predictor.predict(dataset=test_data.input, num_samples=100)
+
+# --- EVALUATION STARTS HERE ---
+
+batches = get_evaluation_data_batches(test_data, forecast_it)
```

Comment:
I think I would prefer an interface more like

```py
evaluator.evalute(predictor, test_data)
```

Reply:
That's better, indeed! [Done](https://github.com/awslabs/gluonts/pull/2267/commits/34e76d814751457721761243ac07b090c299f912).